neo/Consensus/ConsensusService.cs

```diff
@@ -122,8 +122,11 @@ private void CheckCommits()
             if (context.CommitPayloads.Count(p => p?.ConsensusMessage.ViewNumber == context.ViewNumber) >= context.M && context.TransactionHashes.All(p => context.Transactions.ContainsKey(p)))
             {
                 Block block = context.CreateBlock();
-                Log($"relay block: height={block.Index} hash={block.Hash} tx={block.Transactions.Length}");
-                localNode.Tell(new LocalNode.Relay { Inventory = block });
+                if (block != null)
```

Comment:
It can't be `null` because there are already `M` commits.

Reply:
But `EnsureHeader` can return `null`, that's the reason for this check, maybe we can change it now

Reply:
`EnsureHeader` won't return `null` when there are already `M` commits.